build.xml

```diff
@@ -2102,4 +2102,5 @@
   <import file="${build.helpers.dir}/build-rat.xml"/>
   <import file="${build.helpers.dir}/build-owasp.xml"/>
   <import file="${build.helpers.dir}/build-git.xml"/>
+  <import file="${basedir}/.build/build-checkerframework.xml"/>
```

Comment:
@jacek-lewandowski this can be "build.helpers.dir", no? as on the previous lines. build.helpers.dir is alias for basedir/.build